remix-resolve/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "remix-resolve",
```

Comment:
is this name is ok for everyone? i would perhaps prefer `remix-url-resolve` . but i am fine if you are ok wit this too.
cc @GrandSchtroumpf @iurimatias

Reply:
Does it resolve only `import` statement or any other operations based on a url?

Reply:
it's not tied to the solidity `import` statement, it can be used by anyone who want to resolve content of url.

Reply:
I like `remix-url-resolve` or `remix-url-resolver` much better. It's longer, but at least we really know what it's about.